README.md

```diff
@@ -44,46 +42,321 @@ see https://github.com/ioBroker/ioBroker/wiki/Installation
 
 After that the ioBroker should run and be available in browser under ```http://<ip>:8081/```
 
-### Start ioBroker controller on linux
+### Start ioBroker controller
+
+#### ... on Linux
 
 * run ```./iobroker start``` in the ioBroker directory to start the ioBroker controller in the background
-* watch the logfile ```tail -f log/iobroker.log```
+* watch the logfile ```tail -f log/iobroker.<Date>.log```
 
 or
 
 * run ```node node_modules/iobroker.js-controller/controller.js``` to start the ioBroker controller in foreground and watch the log on console
 
-### Start ioBroker controller on windows
+### ... on windows
 
 * run ```iobroker start``` in the ioBroker directory to start the ioBroker controller in the background console
 * check the logfile ```node_modules/iobroker.js-controller/log/iobroker.log```
 
 or
 
 * run ```node node_modules/iobroker.js-controller/controller.js``` to start the ioBroker controller in foreground and watch the log on console
-### Install Redis (Optional)
-- Linux from here: https://redis.io/download
-- Windows from here: https://github.com/MicrosoftArchive/redis/releases
+
+## Configuration
+
+The main configuration is stored in iobroker-data/iobroker.json
```

Comment:
What does a user do with this information?

Reply:
goal for these infos is "teckie user"/dev ... not really end users ... so he may look into it ... also becaus ethe file content is referenced in other areas of this document :-)